packages/server/src/converse/service.ts

```diff
@@ -33,6 +40,37 @@ export class ConverseService extends Service {
     }
 
     const incomingId = this.incomingIdCache.get(message.id)
+    await this.distributed.publish(`converse/${incomingId}`, {
+      cmd: ConverseCmds.Message,
+      data: { message, incomingId }
```

Comment:
Do we really need to send the incomingId considering that the subscriber already knows it?

Reply:
This is builtin in dispatcher